packages/components/src/internal/components/domainproperties/dataclasses/DataClassDesigner.spec.tsx

```diff
@@ -31,14 +30,17 @@ beforeAll(() => {
 });
 
 describe('DataClassDesigner', () => {
-    test('default properties', () => {
+    test('default properties', async () => {
         const form = <DataClassDesigner {...BASE_PROPS} />;
 
-        const tree = renderer.create(form);
+        const tree = mount(form);
```

Comment:
looks like the related snapshots had > 31,000 row changes! whoa. I assume it is because of this change to mount(). I can't see the >31,000 row changes here in the github UI. I assume those snapshots look pretty drastically different?

Reply:
Updated these and the other DomainForm users to only use shallow with snapshots